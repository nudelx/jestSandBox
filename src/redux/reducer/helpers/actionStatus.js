 const actionStatus = {
  FETTCHING: 'FETTCHING',
  DONE: 'DONE',
  FAIL: 'FAIL',
}

export default actionStatus
