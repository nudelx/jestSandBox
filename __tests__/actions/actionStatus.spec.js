import actionStatus from  '../../src/redux/action/actionsType'
import enzymeSerializer from 'enzyme-to-json/serializer';


describe('you should not change !', () => {
  it('actionsType', () => {
    expect.addSnapshotSerializer(enzymeSerializer)
    expect(actionStatus).toMatchSnapshot()
  })
})
