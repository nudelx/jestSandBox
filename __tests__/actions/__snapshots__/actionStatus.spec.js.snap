// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`you should not change ! actionsType 1`] = `undefined`;
